OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -951,6 +951,19 @@ private static void HandleSuffixPronoun(ref StringBuilder formattedString, ReadO
             return null;
         }
 
+        public static ProcStatus? ModulusModulus(DMProcState state) {
+            DreamValue second = state.Pop();
+            DreamValue first = state.Pop();
+
+            if (first.Type == DreamValue.DreamValueType.Float && second.Type == DreamValue.DreamValueType.Float) {
+                state.Push(ModulusModulusValues(first, second));
+            } else {
+                throw new Exception("Invalid modulusmodulus operation on " + first + " and " + second);
+            }
```

Comment:
`ModulusModulusValues()` already does the type check and exception
```suggestion
            state.Push(ModulusModulusValues(first, second));
```